packages/vitest/src/node/plugins/index.ts

```diff
@@ -142,6 +142,8 @@ export async function VitestPlugin(
                   ?? viteConfig.test?.isolate,
               },
             },
+            root: testConfig.root ?? viteConfig.test?.root,
+            deps: testConfig.deps ?? viteConfig.test?.deps,
```

Comment:
I add these here to so that `root` and `deps` from inline test config can be referenced from `VitestOptimizer` plugin.